rspirv/sr/autogen_types.rs

```diff
@@ -16,13 +16,8 @@
 //   external/spirv.core.grammar.json.
 // DO NOT MODIFY!
 
-#[derive(Clone, Debug, Eq, PartialEq)]
-pub struct Function {
```

Comment:
Technically the spec allows you to put references to functions into composite types, however, the entire CFG is static and it is impossible to call them so it's completely useless as far as I can tell (and probably unsupported by drivers). This means we could leave Function outside the Type enum, as long as we are reasonably confident no-one is going to complain about that omission and that the spec won't change to accommodate that, which we probably can be because it would be a huge change to GPU architectures to allow that.